apiProperties.json

```diff
@@ -0,0 +1,23 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "API-Schl\u00fcssel",
+          "description": "\"API-Schl\u00fcssel\" f\u00fcr diese API",
+          "tooltip": "Stellen Sie \"API-Schl\u00fcssel\" bereit.",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#da3b01",

```

Comment:
@jforscht this iconBrandColor is what we have for our Independent Publisher connectors, and it does not match the background color for the icon you provided. 
"#da3b01" is red color.
Please [make sure that the icon meets all the requirements,](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector) that it’s size is 230x230, icon background is not white or transparent and it matches the iconBrandColor property in apiProperties.json file)